coordinator/src/db/positions.rs

```diff
@@ -315,8 +315,6 @@ impl Position {
             ))
             .execute(conn)?;
 
-        ensure!(affected_rows > 0, "Could not set position to open");
```

Comment:
Good catch, that doesn't need to change.

Reply:
Oh now I see, we can't remove `ensure!` here because this function returns a `QueryResult` now. Also I don't think we have to be so strict here. If we can't update the position to `Open` I wouldn't return an error. But I can certainly add this check again if you think it's useful.

Reply:
I opted to return the affected rows instead of failing immediately, so the caller can decide what to do if there was nothing updated.